### PR TITLE
Address issues in forks_test

### PR DESCRIPTION
**Pull request by j16r**
1. Ensure the forks_test tests live code
2. Use curl in docker so that `forks_test` can be run on OS X
3. Reduce number of checks for uncles on a run (once per run vs once per task)
4. Fix an issue where resuming a task with insufficient confirmations on reconnection to the blockchain would skip the confirmation check
5. Fix SHA and VERSION details in final binary
6. Silence warning about no server certificates for fork test chainlink

[Fixes #169658935]
[Fixes #169659501]
[Fixes #169632066]
[Fixes #169659515]
[Finishes #169659528]

### CODE REVIEW

File: core/services/run_manager.go [hunk 1]
```diff
@@ -224,19 +222,19 @@ func (jm *runManager) Create(
 	}
 
 	run.RunRequest = *runRequest
+	currentTaskRun := run.TaskRuns[0]
```
Comment (coventry) [inline core/services/run_manager.go]: Should this be `currentTaskRun := &run.TaskRuns[0]`? As it is, I think 'currentTaskRun` is a copy, which has no effect.

File: core/services/runs.go [hunk 2]
```diff
@@ -26,11 +26,23 @@ func MeetsMinimumPayment(
 
 func validateMinimumConfirmations(run *models.JobRun, taskRun *models.TaskRun, currentHeight *models.Big, txManager store.TxManager) {
 	updateTaskRunConfirmations(currentHeight, run, taskRun)
+
 	if !meetsMinimumConfirmations(run, taskRun, run.ObservedHeight) {
-		logger.Debugw("Pausing run pending confirmations", []interface{}{"run", run.ID.String(), "required_height", taskRun.MinimumConfirmations}...)
```
Comment (coventry) [inline core/services/runs.go]: Why remove the report of the required height? I found that pretty useful, when I was working on the fork test.

Comment (j16r) [reply core/services/runs.go]: It's on the next line.

File: core/services/run_manager.go (hunk 1 above)
Comment (j16r) [reply core/services/run_manager.go]: You're right, removed.

File: core/services/run_executor.go [hunk 3]
```diff
@@ -68,17 +70,19 @@ func (je *runExecutor) Execute(runID *models.ID) error {
 			return err
 		}
 
-		if run.Status.Finished() {
-			logger.Debugw("All tasks complete for run", run.ForLogger()...)
+		if !run.Status.Runnable() {
```
[the remark this answers is not in the document]
Comment (j16r) [reply core/services/run_executor.go]: I don't think it can hurt...

**Pull request merged**